### PR TITLE
routes

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-3-2

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
i like the way keep all routes in one place, good structure of break down component

File: my-app/src/Routes/Routespath.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <Route path='/products' element={<Products />}  />
+  <Route path='/wishList' element={<Wishlist />}  />
+  <Route path='/login' element={<Login />}  />
+  <Route path='/signup'  element={<Signup />} />
+  <Route path='/cart'  element={<Cart/>} />
+  </Routes>
+    </div>
+  )
+}
+
```
Comment (TheAmlanSaikia) [inline my-app/src/Routes/Routespath.jsx:19]: I didn't know we can make a route as a component.

File: my-app/src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+              <li>Standard</li>
+          </ul>
+      </div>
+      <div className=' p-s'>
+          <ul className='fotter-content'>
+              <li className='f-m font-xl '>Playx-store</li>
+              <li>Products</li>
+              <li>Account</li>
+              <li>WishLisht</li>
+              <li>Order</li>
```
Comment (TheAmlanSaikia) [inline my-app/src/components/Footer/Footer.jsx:34]: Try to map over. DRY

File: my-app/src/App.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
```
Comment (saicharan-gouru) [inline my-app/src/App.css:11]: You can add media queries at the end.

**Review by TheAmlanSaikia — APPROVED** [review]
Left a few suggestions. Node modules shouldn't be pushed to GitHub, as far as I know. If in future you install any package lots of changes will be shown. Please confirm it.

File: my-app/src/App.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+.brand-description {
+  font-size: 1.4rem;
+}
+
+.links {
+  display: flex;
+}
+
+.links a {
```
Comment (saicharan-gouru) [inline my-app/src/App.css:48]: You can give a class or id name instead directly using tag name.

File: my-app/src/Routes/Routespath.jsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {Routes, Route} from "react-router-dom";
+import {Home,Login,Signup,Cart,Products,Wishlist} from "../pages/index"
```
Comment (saicharan-gouru) [inline my-app/src/Routes/Routespath.jsx:3]: No need to mention "/index" here.

File: my-app/src/components/Footer/Footer.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+.foot{
```
Comment (saicharan-gouru) [inline my-app/src/components/Footer/Footer.css:1]: You can use meaningful names instead of shortcuts. EX: footer

**Review by saicharan-gouru — APPROVED** [review]
-Good that you have added gif for reference.
-You can also add description while creating the PR.
-Consider the suggested comments. Everything looks good to merge.

File: my-app/src/Routes/Routespath.jsx (hunk 1 above)
Comment (aniketsingh13) [reply my-app/src/Routes/Routespath.jsx:19]: yes we can do that

File: my-app/src/components/Footer/Footer.jsx (hunk 2 above)
Comment (aniketsingh13) [reply my-app/src/components/Footer/Footer.jsx:34]: ok will do that

File: my-app/src/App.css (hunk 4 above)
Comment (aniketsingh13) [reply my-app/src/App.css:48]: will change that

File: my-app/src/App.css (hunk 3 above)
Comment (aniketsingh13) [reply my-app/src/App.css:11]: its on default file from backend will delete it

File: my-app/src/App.css (hunk 4 above)
Comment (aniketsingh13) [reply my-app/src/App.css:48]: its on default file from backend will delete it

File: my-app/src/Routes/Routespath.jsx (hunk 5 above)
Comment (aniketsingh13) [reply my-app/src/Routes/Routespath.jsx:3]: ok will change that

File: my-app/src/components/Footer/Footer.css (hunk 6 above)
Comment (aniketsingh13) [reply my-app/src/components/Footer/Footer.css:1]: will change that

**Pull request merged**